runners/direct-java/pom.xml

```diff
@@ -262,7 +262,15 @@
       </exclusions>
       <scope>test</scope>
     </dependency>
-    
+
+    <!-- Required for UnboundedReadFromBoundedSourceTest -->
+    <dependency>
+      <groupId>com.esotericsoftware.kryo</groupId>
+      <artifactId>kryo</artifactId>
+      <version>2.21</version>
```

Comment:
why is this needed at all -- isn't it picked up from the fact that this depends on runner-core?